__tests__/ordCdsrc.test.js

```diff
@@ -1,26 +1,19 @@
 const cds = require("@sap/cds");
-const ord = require("../lib/ord");
 const path = require("path");
 
-// Mock the @sap/cds module
-jest.mock("@sap/cds", () => {
-    const path = require("path");
-    let cds = jest.requireActual("@sap/cds");
-    cds.root = path.join(__dirname, "bookshop");
-
-    return cds;
-});
-
-jest.mock("../lib/date", () => ({
-    getRFC3339Date: jest.fn(() => "2024-11-04T14:33:25+01:00")
-}));
-
-
 describe("Tests for default ORD document when .cdsrc.json is present", () => {
-    let csn;
+    let csn, ord;
```

Comment:
what is the benefit of loading ord in beforeAll?

Reply:
You can check by move up the import statement. Test will not work. It has to do with the why how the code is written. We first need to mock something and then to import it, if we want our tests to work.